main.py

```diff
@@ -88,25 +88,52 @@ def create(self, email, password):
         self.main()
 
     def main(self):
-        while True:
-            print(
-            """
-            Main Menu
-            1. Emergency Alerts
-            2. Energy Efficiency
-            """
-            )
-            try:
-                choice = int(input("\n\nChoose option: "))
-                break
-            except:
-                print("\nInvalid choice. Try again.")
-                continue
+        try:
+            while True:
+                print(
+                """
+                Main Menu
+                1. Emergency Alerts
+                2. Energy Efficiency
+                """
+                )
+                try:
+                    choice = int(input("\n\nChoose option: "))
+                    break
+                except:
+                    print("\nInvalid choice. Try again.")
+                    continue
+
+            print(choice)
+        
+            if(choice):
+                if(choice == 1):
+                    self.alert_function_1()
 
-        print(choice)
+                else:
+                    print('Not one of our options. Exiting.')
+        except:
+            print("Error")
```

Comment:
This error message is not descriptive.
```suggestion
            print("Invalid input. Please try again")
```